backend/resources/akvo/lumen/migrations/tenants/012-plans.up.sql

```diff
@@ -0,0 +1,118 @@
+CREATE TABLE tier (
+     id text PRIMARY KEY DEFAULT gen_random_uuid()::text,
+     title text NOT NULL UNIQUE CHECK (title IN ('lite','standard','pro','enterprise'))
```

Comment:
Will migrations be more difficult if the current set of valid values are listed as a constraint?